grml-debootstrap

```diff
@@ -13,7 +13,14 @@ error_handler() {
    last_bash_command="$BASH_COMMAND"
    echo "Unexpected non-zero exit code $last_exit_code in ${BASH_SOURCE[*]} at line ${BASH_LINENO[*]} detected!
 last bash command: $last_bash_command"
-   ## Check if "bailout" function is already available.
+   if [ -r "$MNTPOINT/debootstrap/debootstrap.log" ] && \
+          [ -s "$MNTPOINT/debootstrap/debootstrap.log" ] ; then
+       einfo "Presenting last ten lines of debootstrap.log:"
+       tail -10 "${MNTPOINT}"/debootstrap/debootstrap.log
+       einfo "End of debootstrap.log"
+   fi
+   ## Check if "bailout" function is available.
+   ## This is not the case in chroot-script.
```

Comment:
hm, I don't understand why this change ends up in this clean_chroot feature/commit, is the github web UI showing something unexpected? o_

Reply:
I am not sure where you saw that. But GitHub says `Outdated`". I cannot see any change related to / near `## Check if "bailout" function is available.` in a local git diff.